pkg/otp/code.go

```diff
@@ -133,10 +135,14 @@ func (o *Request) Issue(ctx context.Context, retryCount uint) (string, string, s
 			LongExpiresAt: o.LongExpiresAt,
 			IssuingUserID: issuingUserID,
 			IssuingAppID:  issuingAppID,
+			UUID:          o.UUID,
 		}
 		// If a verification code already exists, it will fail to save, and we retry.
-		if err := o.DB.SaveVerificationCode(&verificationCode, o.MaxSymptomAge); err != nil {
```

Comment:
I think this is a bug. This `err` is scoped to this if statement, but when we break we `return "", "", "", err` which is unset in that scope